### PR TITLE
10.14.0.324

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+<tr>
+<td>Authentication</td>
+<td>The performance of the first user requests sent after node restart has been improved. Prior to this change, there was an issue in rare cases where the number of all global roles in a tenant was close to but not exceeding 100, and many thousands of devices were concurrently trying to authorize MQTT connections on the restarted node.</td>
+<td>MTM-52049</td>
+</tr>
+
+<tr>
+<td>Core platform</td>
+<td>By adding or removing an external identifier of type <code>c8y_Serial</code> to/from the monitored device, the device connection monitoring is notified accordingly to start or stop with the specified external identifier.</td>
```
Comment (rssag) [inline content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md:31]: this one is odd, had to read it several time to grasp what was meant.
"the monitored device" -> "a monitored device"
what does "start or stop with" mean?

Comment (BeateRixen) [reply content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md:31]: Same for me, I hoped that you get it better than I do :-). I have asked the developer in the ticket to rephrase the second half of the sentence.

Comment (BeateRixen) [inline content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md:31]:
```suggestion
<td>For devices which are monitored by the device availability monitoring functionality, removing the external ID with type <code>c8y_Serial</code> from the device will stop monitoring ID while adding a new external ID with type <code>c8y_Serial</code> will start monitoring this ID.
</td>

```

Comment (BeateRixen) [reply content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md]: @rssag Better :-)?

File: content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<tr>
+<td>Authentication</td>
+<td>The performance of the first user requests sent after node restart has been improved. Prior to this change, there was an issue in rare cases where the number of all global roles in a tenant was close to but not exceeding 100, and many thousands of devices were concurrently trying to authorize MQTT connections on the restarted node.</td>
+<td>MTM-52049</td>
+</tr>
+
+<tr>
+<td>Core platform</td>
+<td>For devices which are monitored by the device availability monitoring functionality, removing the external ID with type <code>c8y_Serial</code> from the device will stop monitoring ID while adding a new external ID with type <code>c8y_Serial</code> will start monitoring this ID.
```
Comment (rssag) [inline content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md:31]:
```suggestion
<td>For devices which are monitored by the device availability monitoring functionality, removing the external ID with type <code>c8y_Serial</code> from the device stops the monitoring of this ID while adding a new external ID with type <code>c8y_Serial</code> starts the monitoring of this ID.

```

**Review by rssag — APPROVED** [review]
approved, but one suggestion

**Pull request merged**